apps/site/src/app/(home)/sections/FAQ/FAQ.tsx

```diff
@@ -0,0 +1,31 @@
+import { getQuestions } from "./getQuestions";
+import FAQAccordion from "./FAQAccordion";
+
+import { PortableText } from "@portabletext/react";
+import styles from "./FAQ.module.scss";
+
+const FAQ = async () => {
+	const questions = await getQuestions();
+	const faq = questions[0]["faqs"].map(({ _key, question, answer }) => ({
+		_key,
+		question: <strong className="w-10/12">{question}</strong>,
+		answer: <PortableText value={answer} />,
+	}));
+
+	return (
+		<section
+			className={`${styles.container} rounded-[500rem/20rem] py-24 flex justify-center bg-no-repeat bg-cover bg-center bg-top`}
```

Comment:
Issue: Slightly unrelated but if you could add some padding below the mentor and volunteer section to make the spacing between that section and the FAQs more even, that would be great.